content/pub-sub/advanced.textile

```diff
@@ -905,54 +905,174 @@ h3(#idempotency). Idempotent publishing
 
 Idempotency ensures that multiple publishes of the same message cannot result in duplicate messages.
 
-It is possible that a client publishing a message using the REST interface may not receive acknowledgement of receipt from Ably, due to issues such as network failure outside of Ably's control. Clients will internally attempt to re-publish messages in these instances.
-
 When idempotent publishing is enabled, the Ably SDK will internally assign a unique ID to each message which ensures that subsequent retry attempts cannot result in duplicate messages. Idempotent publishing is enabled by default in all latest Ably SDKs. It can be disabled by setting the @idempotentRestPublishing@ "@ClientOptions@":/docs/api/rest-sdk#client-options to @false@.
```

Comment:
Do we know how many people are using the SDK versions that don't have it enabled by default? Is it worth just updating this to say:

```
Idempotent publishing is enabled by default, the Ably SDK will internally assign a unique ID to each message which ensures that subsequent retry attempts cannot result in duplicate messages. It can be disabled by setting the @idempotentRestPublishing@ "@ClientOptions@":/docs/api/rest-sdk#client-options to @false@.
```